src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -117,6 +130,19 @@
     margin-top: var(--pf-c-dropdown__c-divider--MarginTop);
     margin-bottom: var(--pf-c-dropdown__c-divider--MarginBottom);
   }
+
+  .pf-c-avatar {
```

Comment:
If this is just for positioning, can we create a `pf-c-dropdown__image` (or whatever) element that's agnostic to what is placed in it? Then other images/icons could be used here and these rules have less specificity and there's no dependency created between the dropdown and avatar.